src/components/App/ExpressRecorder.tsx

```diff
@@ -757,33 +797,33 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
                         </button>
                     )}
                     {showUploadUI &&
-                        !doRecording &&
-                        recordedBlobs.length > 0 &&
-                        !this.uploadedOnce && (
-                            <div className={`${styles["express-recorder__bottom"]}`}>
-                                <button
-                                    className={`xr_btn xr_btn__download ${styles["bottom__btn"]} ${styles["btn__clear"]} ${styles["btn__download"]} `}
-                                    onClick={this.saveFile}
-                                    tabIndex={0}
-                                >
-                                    {this.translator.translate("Download a Copy")}
-                                </button>
-                                <button
-                                    className={`xr_btn xr_btn__reset ${styles["bottom__btn"]} ${styles["btn__clear"]}`}
-                                    onClick={this.recordAgain}
-                                    tabIndex={0}
-                                >
-                                    {this.translator.translate("Record Again")}
-                                </button>
-                                <button
-                                    className={`xr_btn xr_btn-primary xr_btn__save ${styles["bottom__btn"]} ${styles["btn__save"]}`}
-                                    onClick={this.initiateUpload}
-                                    tabIndex={0}
-                                >
-                                    {this.translator.translate("Use This")}
-                                </button>
-                            </div>
-                        )}
+                    !doRecording &&
+                    (recordedBlobs.length || screenRecordedBlobs.length) &&
+                    !this.uploadedOnce ? (
+                        <div className={`${styles["express-recorder__bottom"]}`}>
+                            <button
+                                className={`xr_btn xr_btn__download ${styles["bottom__btn"]} ${styles["btn__clear"]} ${styles["btn__download"]} `}
+                                onClick={this.saveFile}
+                                tabIndex={0}
```

Comment:
```suggestion
          aria-label={translate("Click here to download a copy of your video")}
```
why do you use `tabIndex`? by default, it's set to `0` anyway.
please add `aria-label` attribute for better accessibility.

please go over all other buttons as well.

Reply:
you right, no need for tabindex.
no need for aria label on buttons - it's read the text on the button